.github/workflows/verify_consumer_pacts.yml

```diff
@@ -319,8 +319,8 @@ jobs:
                           -e JANITOR_CLIENT_CREDENTIAL_FILE_PATH="" \
                           -e JANITOR_TRACK_RESOURCE_PROJECT_ID="" \
                           -e JANITOR_TRACK_RESOURCE_TOPIC_ID="" \
-                          sbtscala/scala-sbt:openjdk-17.0.2_1.7.2_2.13.10 \
-                          sbt "set scalafmtOnCompile := false" "project pact4s" "testOnly *SamProviderSpec"
+                          sbtscala/scala-sbt:eclipse-temurin-jammy-17.0.10_7_1.10.2_2.13.15 \
+                          bash -c "git config --global --add safe.directory /working/sam && sbt \"set scalafmtOnCompile := false\" \"project pact4s\" \"testOnly *SamProviderSpec\""
```

Comment:
here and elsewhere, set the `scala-sbt` docker image to the version that uses sbt 1.10.2 and Scala 2.13.15.

The docker image upgrade must have updated its version of git, because it started running into the "dubious ownership" error; so, I also added the incantation to set the git `safe.directory`.